official/utils/data/file_io_test.py

```diff
@@ -28,6 +28,7 @@
 # pylint: enable=wrong-import-order
 
 from official.utils.data import file_io
+from official.utils.misc import keras_utils
```

Comment:
totally optional but i just realized "is_v2_0" check should probably not be in a util called keras_utils :)

Reply:
Yeah, I will try to refactor that later.  I move it here to share.  It would be nice if TensorFlow provided this as a feature.  Not fixing in this PR.